src/js/themes/__tests__/__snapshots__/theme-test.js.snap

```diff
@@ -1579,7 +1579,7 @@ exports[`Grommet hpe theme 1`] = `
   box-sizing: border-box;
   max-width: 100%;
   background-color: #01A982;
-  color: #EEEEEE;
+  color: #FFFFFF;
```

Comment:
Can you verify what is this color change impacting? I see too many similar changes across the snapshot and we need to know the root cause for this change.